lib/src/lifecycle_module.dart

```diff
@@ -234,11 +246,14 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
       return null;
     }
 
+    _logger.finest(
```

Comment:
Open to killing this log, but it's pretty nice to have in your console.

Reply:
Only seconds granularity? That's rather pessimistic 😉 

Haha but for real, I think more precision on this might be nice ("took 1.2s"/"took 1200ms"/etc.)

Reply:
Totally agree, I just started off with something useful for debugging but I can clean this up.